plover/gui_qt/dictionaries_widget.py

```diff
@@ -91,6 +125,16 @@ def __init__(self, *args, **kwargs):
         self.menu_AddDictionaries.addAction(_(
             _('New dictionary'),
         )).triggered.connect(self._create_new_dictionary)
+        # Save menu.
+        self.menu_SaveDictionaries = QMenu(self.action_SaveDictionaries.text())
+        self.menu_SaveDictionaries.setIcon(self.action_SaveDictionaries.icon())
+        self.menu_SaveDictionaries.addAction(_(
+            _('Create a copy of each dictionary'),
```

Comment:
... double gettext call?

Reply:
Yeah... copying from above... o.O

Reply:
(side note: I didn't notice that comment being posted being making the suggested change below. But I'm sure that I did delete the comment above...)